README.md

```diff
@@ -1,4 +1,4 @@
-# darts: Easy manipulation and forecasting of time series
+# Darts: Easy manipulation and forecasting of time series.
```

Comment:
How about "Darts: Time Series Made Easy in Python" ?

Reply:
Did you have a look at what I put in the related thread on slack ? Otherwise I will make that change :)

Reply:
Darts: Time Series Made Easy in Python